src/components/UI/downloads/DownloadsHero.tsx

```diff
@@ -29,61 +39,100 @@ export const DownloadsHero: FC<DownloadsHero> = ({
+              further down the page. If you&apos;re looking to install Geth and/or associated tools
+              via your favorite package manager, please check our installation guide.
+            </Text>
+          </Stack>
 
-        <Text mb={4}>
-          You can download the latest 64-bit stable release of Geth for our primary platforms below.
-          Packages for all supported platforms, as well as develop builds, can be found further down
-          the page. If you&apos;re looking to install Geth and/or associated tools via your favorite
-          package manager, please check our installation guide.
-        </Text>
+          <Stack
+            py={{ base: 0 }}
+            px={{ base: 8 }}
+            flex={{ base: 'none' }}
+            display={{ base: 'block', lg: 'none' }}
+            order={{base: -1, md: 1}}
+          >
+            <Center>
+              <Image
+                src='/images/pages/gopher-downloads-front-light.svg'
+                alt='Gopher plugged in'
+                w={{ md: 72 }}
+              />
+            </Center>
+          </Stack>
+        </Grid>
 
-        {Object.keys(DOWNLOAD_HEADER_BUTTONS).map((key: string) => {
-          return (
-            <NextLink key={key} href={DOWNLOAD_HEADER_BUTTONS[key].buildURL} passHref>
-              <Button as='a' variant='primary' width={{ base: '100%' }} p={8} mb={4}>
-                <HStack spacing={4}>
-                  <Stack alignItems='center'>
-                    <Image
-                      src={DOWNLOAD_HEADER_BUTTONS[key].image}
-                      alt={DOWNLOAD_HEADER_BUTTONS[key].imageAlt}
-                    />
-                  </Stack>
-                  <Box>
-                    <Text textStyle='downloads-button-label'>
-                      For {DOWNLOAD_HEADER_BUTTONS[key].name}
-                    </Text>
-                    <Text textStyle='downloads-button-label'>geth {currentBuildName}</Text>
-                  </Box>
-                </HStack>
-              </Button>
-            </NextLink>
-          );
-        })}
+        <Grid templateColumns={{ base: 'repeat(1, 1fr)', md: 'repeat(2, 1fr)' }} gap={4} mb={4}>
+          {Object.keys(DOWNLOAD_HEADER_BUTTONS).map((key: string) => {
+            return (
+              <NextLink key={key} href={DOWNLOAD_HEADER_BUTTONS[key].buildURL} passHref>
+                <Button as='a' variant='primary' width={{ base: '100%' }} p={8}>
+                  <HStack spacing={4}>
+                    <Stack alignItems='center'>
+                      <Image
+                        src={DOWNLOAD_HEADER_BUTTONS[key].image}
+                        alt={DOWNLOAD_HEADER_BUTTONS[key].imageAlt}
+                      />
+                    </Stack>
+                    <Box>
+                      <Text textStyle='downloads-button-label'>
+                        For {DOWNLOAD_HEADER_BUTTONS[key].name}
```

Comment:
Font sizes are different on design (https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=1043%3A8883&t=BRNWeIoFi4HUMoPH-0) with platform name bigger (20px on desktop) and build name smaller (14px). They're different on mobile too (16px and 14px).